Tests/ServerTrustPolicyTests.swift

```diff
@@ -373,11 +373,11 @@ class ServerTrustPolicyExplorationSSLPolicyValidationTestCase: ServerTrustPolicy
 
         // When
         let policies = [
-            SecPolicyCreateSSL(true, "test.alamofire.org"),
-            SecPolicyCreateSSL(true, "blog.alamofire.org"),
-            SecPolicyCreateSSL(true, "www.alamofire.org")
+            SecPolicyCreateSSL(true, "test.alamofire.org" as CFString?),
+            SecPolicyCreateSSL(true, "blog.alamofire.org" as CFString?),
+            SecPolicyCreateSSL(true, "www.alamofire.org" as CFString?)
```

Comment:
These can all probably be cast using `as CFString` instead of adding the optional.